packages/sdk/src/ClientSide.ts

```diff
@@ -25,10 +25,10 @@ export class ClientSide {
 			query.code = params
 		} else {
 			query.code = params.code
-			query.item = params.order?.items
+			query.items = params.items
```

Comment:
And now it is broken - did you test it?
This must be `query.item` since backend is expecting this in format of URL with query params: `item[0]=something&item[1]=somehting2`. Now it is `items[0]=something&items[1]=something2`